background/state.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  version: null
```

Comment:
Would be good to introduce some redux-like state object anyway, for couple of stuff or re-implement the whole background script initialization to be in a single place instead of loading bunch of modules that attach some listeners on their own.